src/SDKs/DBforPostgreSQL/Management.DBforPostgreSQL/generate.cmd

```diff
@@ -0,0 +1,7 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
```

Comment:
@rohit-joy add generate.ps1 instead of generate.cmd.
you can use the one you see under compute as an example.
Also make sure you use generate.ps1 to generate you sdk, because that creates a meta data file that is required for your SDK to be merged.

Reply:
Done